mbed_app.json

```diff
@@ -428,6 +428,53 @@
             "storage_tdb_internal.internal_base_address": "(MBED_ROM_START+1024*1024)",
             "storage_tdb_internal.internal_size"        : "(32*1024)",
             "storage.storage_type"                      : "TDB_INTERNAL"
+        },
+        "UHURU_RAVEN": {
+            "target.macros_remove"                      : ["MBEDTLS_CONFIG_HW_SUPPORT"],
+            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS.h\""],
+            "target.network-default-interface-type"     : "WIFI",
+            "target.components_add"                     : ["WIFI_ESP32", "FLASHIAP"],
+            "target.bootloader_img"                     : "bootloader/mbed-bootloader-stm32f767-internal_flash-no_rot-v4.1.2.bin",
+            "target.header_offset"                      : "0x18000",
+            "target.app_offset"                         : "0x18400",
+            "target.restrict_size"                      : "0xE8000",
+            "esp32.wifi-debug"                          : false,
+            "esp32.wifi-en"                             : "PB_12",
+            "esp32.wifi-io0"                            : "PB_13",
+            "esp32.wifi-tx"                             : "PB_14",
+            "esp32.wifi-rx"                             : "PB_15",
+            "esp32.provide-default"                     : true,
+            "update-client.bootloader-details"          : "0x8005d30",
+            "update-client.application-details"         : "(MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS + MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-address"             : "(MBED_ROM_START + (MBED_ROM_SIZE/2))",
+            "update-client.storage-size"                : "((MBED_ROM_SIZE/2) - MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-locations"           : 1,
+            "update-client.storage-page"                : 1,
+            "update-client.firmware-header-version"     : "2",
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP",
+            "storage.storage_type"                      : "TDB_INTERNAL",
+            "storage_tdb_internal.internal_base_address": "(MBED_ROM_START + 32*1024)",
```

Comment:
This probably should be in (MBED_ROM_START + (MBED_ROM_SIZE/2) + MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE) ?